backend/apps/slack/blocks.py

```
@@ -2,20 +2,38 @@
         "text": {"type": "mrkdwn", "text": text},
     }
 
 
 def get_header():
-    """Return the header block."""
+    """
+    Return the header block.
+
+    Returns:
+        list: A list of Slack blocks representing the header.
+    """
```

Ruff (0.8.2)

Multi-line docstring summary should start at the first line

Remove whitespace after opening quotes

(D212)